### PR TITLE
Active website only

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: Backoffice/Context/ContextManager.php [hunk 1]
```diff
@@ -76,7 +77,11 @@ public function getAvailableSites()
 
         if ($token && ($user = $token->getUser()) instanceof GroupableInterface) {
             foreach ($user->getGroups() as $group) {
-                $sites[] = $group->getSite();
+                /** @var SiteInterface $site */
+                $site = $group->getSite();
+                if (!$site->isDeleted()) {
+                    $sites[] = $group->getSite();
+                }
```
Comment (itkg-rboessel) [inline Backoffice/Context/ContextManager.php]: $sites[] = $site;

Comment (itkg-ngilain) [inline Backoffice/Context/ContextManager.php]: +1 @itkg-rboessel

**Comment by nicolasThal** [comment]
@itkg-ngilain @itkg-rboessel update done

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**